gobblin-yarn/src/main/java/gobblin/yarn/GobblinHelixJobLauncher.java

```diff
@@ -121,6 +120,9 @@ public GobblinHelixJobLauncher(Properties jobProps, HelixManager helixManager, F
     this.stateSerDeRunnerThreads = Integer.parseInt(jobProps.getProperty(ParallelRunner.PARALLEL_RUNNER_THREADS_KEY,
         Integer.toString(ParallelRunner.DEFAULT_PARALLEL_RUNNER_THREADS)));
 
+    URI fsUri = URI.create(jobProps.getProperty(ConfigurationKeys.FS_URI_KEY, ConfigurationKeys.LOCAL_FS_URI));
+    this.fs = FileSystem.get(fsUri, new Configuration());
```

Comment:
AFAIK the only thing it was used for was the FileBasedJobLock and rather than pass it in just for that, let the JobLock initialize itself from properties.